test/commons/color/element-is-distinct.js

```diff
@@ -1,271 +1,438 @@
 describe('color.elementIsDistinct', () => {
-  let styleElm;
-  let elementIsDistinct;
+  const elementIsDistinct = axe.commons.color.elementIsDistinct;
+  const fixtureSetup = axe.testUtils.fixtureSetup;
+  const fixture = document.querySelector('#fixture');
+
+  function convertStylePropsToString(props = {}) {
+    return Object.entries(props)
+      .map(([name, value]) => `${name}: ${value}`)
+      .join(';');
+  }
 
-  const fixture = document.getElementById('fixture');
+  function createTestFixture({ target, root, inline } = {}) {
```

Comment:
I think I would prefer for `inline` to be called `targetWrap`. `inline` feels a little vague, especially since I don't think all inline ancestors should be considered (see comments above). A sentence like `returns true if inline adds outline` also messes with my brain a bit, that might just be me though.